packages/vision/src/helpers.js

```diff
@@ -101,6 +106,19 @@ module.exports = apiVersion => {
       options = undefined;
     }
 
+    // If there is no image, throw an exception.
+    if (is.undefined(request.image)) {
+      throw new Error('Attempted to call `annotateImage` with no image.');
+    }
+
+    // If we got a filename for the image, open the file and transform
+    // it to content.
+    if (request.image.source && request.image.source.filename) {
+      request.image = {
+        content: fs.readFileSync(request.image.source.filename),
```

Comment:
We should defer to performing I/O asynchronously as much as possible, as this will block the user's whole app.